index.html

```diff
@@ -2096,8 +2099,8 @@ <h2>Resource Integrity</h2>
 values.
         </p>
         <p>
-JSON-LD context authors are expected to add `digestMultibase` to contexts that will 
-be used in documents that refer to other resources and include an associated 
+JSON-LD context authors are expected to add `digestMultibase` to contexts that will
+be used in documents that refer to other resources and include an associated
 content integrity hash. For example, the [[[VC-DATA-MODEL-2.0]]]
 context (`https://www.w3.org/ns/credentials/v2`) includes it.
```

Comment:
I'm not sure whether we're blurring `hash` and `digest` and/or maybe mis-referring to one or the other. I may also have fumbled the data cite markup, but hopefully the suggestion is clear enough to repair.
```suggestion
JSON-LD context authors are expected to add `digestMultibase` to contexts that will
be used in documents that refer to other resources and to include an associated
cryptographic digest. For example, the [[[VC-DATA-MODEL-2.0]]] refers to
context (`https://www.w3.org/ns/credentials/v2`) which includes
`digestMultibase`, and the [[[VC-DATA-MODEL-2.0]]] includes
<data cite="VC-DATA-MODEL-2.0/#base-context">the hexadecimal encoded
SHA2-256 digest value</data> of that context document.
```

Reply:
Yes, we were unnecessarily blurring the lines. I have unblurred them by using "digest" or "cryptographic digest" and made the modification into your change suggestion.